### PR TITLE
Fixed golint issues in maya/pkg/snapshot/v1alpha1/snapshot_template_e…

### PR DESCRIPTION
**Pull request by rockydcoder**
…ngine.go

Signed-off-by: Rakesh Sarangi <rakesh.rockyrocks.xxx@gmail.com>

**Fixed golint issues in maya/pkg/snapshot/v1alpha1/snapshot_template_engine.go**:

`fixes #2201`

[Issue](https://github.com/openebs/openebs/issues/2201)

### CODE REVIEW

File: pkg/snapshot/v1alpha1/snapshot_template_engine.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This file has snapshot specific implementation of cas template engine
+// Package snapshot has snapshot specific implementation of cas template engine
```
Comment (princerachit) [inline pkg/snapshot/v1alpha1/snapshot_template_engine.go]: This change is not required. Just add one empty line here.

Comment (rockydcoder) [reply pkg/snapshot/v1alpha1/snapshot_template_engine.go]: Done

File: pkg/snapshot/v1alpha1/snapshot_template_engine.go [hunk 2]
```diff
@@ -15,6 +15,7 @@ limitations under the License.
 */
 
 // This file has snapshot specific implementation of cas template engine
```
Comment (sonasingh46) [inline pkg/snapshot/v1alpha1/snapshot_template_engine.go]: Can you please delete this comment line.

Comment (rockydcoder) [reply pkg/snapshot/v1alpha1/snapshot_template_engine.go]: Done

File: pkg/snapshot/v1alpha1/snapshot_template_engine.go [hunk 3]
```diff
@@ -14,7 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This file has snapshot specific implementation of cas template engine
```
Comment (princerachit) [inline pkg/snapshot/v1alpha1/snapshot_template_engine.go]: Revert this change. Add a newline after the above comment.
